### ISSUE TITLE
 unfair abandonment record match Dota 2

### ISSUE THREAD

**Issue by uriel159** [ValveSoftware/Dota2-Gameplay#7945]
### Description

I had internet connection and hardware problems near the min 30 of the game, I was able to reconnect but because of other players who unpaused the game, it counted me as quitting unfairly affecting my score behavior and getting a time penalty and mmr discount.

I consider it somewhat unfair when I tried by all means to reconnect and succeeded in the end, so that I am penalized for abandonment.

Please your support so that this game is not considered as abandonment and leave the consequent penalties without effect, thank you.

### Example Match ID (and possibly Timestamp)

match id: 7104197103

### Screenshots

![reconnect](https://user-images.githubusercontent.com/130594938/231544908-c8004930-47a0-4960-8cd5-f56879a48f9f.jpg)

**Comment by TheUnholyWrath** [comment]
This isn't the kind of issue that is dealt with here, no one is going to respond to this issue. There's also nothing wrong happening here. You disconnected and didn't reconnect in time, it sucks that it was out of your control but this still impacts the other 9 people in the game. Theres no sense in holding out for false hope, you should close this issue.

**Comment by bkuepfer-revcomm** [comment]
I agree with @TheUnholyWrath , this issue does not belong here, it just clutters the issues page even more, hiding actual bugs. 
This issue should be closed.
